### DISCUSSION TITLE
Cinema mod with Storyboard disabled does not warn about rapidly flashing colors

### DISCUSSION THREAD

**Discussion by JacksonChen666** [ppy/osu#23685] [Q&A]
### Type

Game behaviour

### Bug description

A beatmap with the rapidly flashing colors warning is only shown if storyboard/video is enabled (good feature).
However, the Cinema mod basically forces that option on.

So when the storyboard/video option is disabled but you have used the Cinema mod, osu! will not warn you about the rapidly flashing colors.

You are warned if you have the storyboard/video option enabled regardless of the Cinema mod.

([Beatmap](https://osu.ppy.sh/beatmapsets/470977#osu/1006822))

### Version

2023.513.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/11582735/database.log)
[input.log](https://github.com/ppy/osu/files/11582736/input.log)
[network.log](https://github.com/ppy/osu/files/11582737/network.log)
[performance-audio.log](https://github.com/ppy/osu/files/11582738/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/11582739/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/11582741/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/11582742/performance-update.log)
[performance.log](https://github.com/ppy/osu/files/11582743/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11582744/runtime.log)

**Comment by peppy** [comment]
@JacksonChen666  Do you have epilepsy?

(Switching to discussion because I am not yet certain on the future of this feature)

**Reply by JacksonChen666** [reply]
I do not have epilepsy, I just noticed the warning was missing in certain cases.

**Reply by JacksonChen666** [reply]
Given that the warning is now just a thing on the top left and the bug still exists, will there be reconsiderations to fix this?
